pages/querying/clauses/alter.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: ALTER clause
+description: Learn how to alter existing definitions of enums.
+---
+
+import { Callout } from 'nextra/components'
+
+# ALTER clause
+
+The `ALTER` clause is used to alter existing definitions of enums.
```

Comment:
s/to alter/to change
Don't use a term to define itself

Reply:
I've changed this while I was working on another update for this PR.